File: Lanche.py
cod, qtd = input().split()
cod = int(cod)
qtd = float(qtd)

if cod == 1:
    preco = 4.00 * qtd
elif cod == 2:
    preco = 4.50 * qtd
elif cod == 3:
    preco = 5.00 * qtd
elif cod == 4:
    preco = 2.00 * qtd
elif cod == 5:
    preco = 1.50 * qtd
print("Total: R$ {:.2f}".format(preco))

File: Lanche.js
const fs = require("fs");
const r = fs.readFileSync(0, "utf8");
const line  = r.split(" ");
let cod = parseInt(line[0]);
let qtd = parseInt(line[1]);
let total = 0;
switch (cod) {
    case 1:
        total = qtd * 4.00;
        break;
    case 2:
        total = qtd * 4.50;
        break;
    case 3:
        total = qtd * 5.00;
        break;
    case 4:
        total = qtd * 2.00;
        break;
    case 5:
        total = qtd * 1.50;
        break;
}
console.log("Total: R$ " + total.toFixed(2));